package-lock.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "cortex-sdk",
-  "version": "0.0.1",
+  "name": "@cortexclick/cortex",
+  "version": "0.0.3",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
-      "name": "cortex-sdk",
-      "version": "0.0.1",
-      "license": "ISC",
+      "name": "@cortexclick/cortex",
```

Comment:
I don't think this is correct... Not sure why my package-lock.json ended up in this state...

Reply:
I think this is actually correct. We probably just didn't run `npm install` before publishing the last version of the package.

Reply:
Ohhh great. I was thinking my repo was in some borked state and was researching how to revert a file from a commit :) I actually avoid checking in this file twice before cuz I didn't think my change was correct :)